### PR TITLE
Feature/rest cleanup

### PR DESCRIPTION
**Pull request by sci-burns**
Use the gradle REST plugin to make: getConflicts, merge, mergeStatus, createRestorePoint, revertToLastRestorePoint.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -42,87 +48,162 @@ task exportProduct{
...
-	doLast {
-		String rp = createRP(SOR_URL);
-		def rpFile = new File("restorepoint.txt");
-		rpFile.write(rp)
+task createRestorePoint(type: org._10ne.gradle.rest.RestTask) {
+	group 'Pega'
+	description 'Create restore point.'
+
+	def targetURL = findProperty("targetURL")
+	uri = "${targetURL}/prweb/PRRestService/SystemManagement/V2/RestorePoint"
```
Comment (petehayes) [inline build.gradle]: Can we leave the /prweb/ off the URL?  Not a huge deal but that part seems more variable depending on the deployment and should be injected into scripts.

Comment (sci-burns) [reply build.gradle]: Yeah, I think you are right - just so I am clear, you mean make that part of the "targetURL". Right?

**Comment by petehayes** [comment]
Looks good to me

**Review by linoypega — APPROVED** [review]
This looks great. I am going to merge these changes.

**Pull request merged**